h2o-core/src/main/java/water/network/SecurityUtils.java

```diff
@@ -68,7 +76,9 @@ public static SSLCredentials generateSSLPair(String passwd, String name, String
     }
 
     public static SSLCredentials generateSSLPair() throws Exception {
-        StoreCredentials jks = generateKeystore(passwordGenerator(16));
+        Path temp = Files.createTempDirectory("h2o-internal-jks-" + Long.toString(System.nanoTime()));
+        temp.toFile().deleteOnExit();
```

Comment:
This will most likely not work because the directory will not be empty.

I tried it on my mac and the file was preserved even after the JVM exited

Reply:
@michalkurka are you sure the JVM exited properly? I'm deleting the file that is created in that directory here https://github.com/h2oai/h2o-3/pull/1691/files#diff-4867aa333cb6f2e4c8211132ba4f0920R53

Reply:
@michalkurka any update on this? Or can I merge this in?

Reply:
one last ping @michalkurka

Reply:
Sorry, must have made a mistake in testing. I take the comment back.